### ISSUE TITLE
Function libxml_disable_entity_loader() is deprecated in...\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Reader\Security\XmlScanner.php on line 67

### ISSUE THREAD

**Issue by TheRealMajaba** [PHPOffice/PhpSpreadsheet#2125]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Work like it did prior to PHP 8.0

### What is the current behavior?
Throws errors and warnings but seems to still be functional

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

// Create new Spreadsheet object
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();

// add code that show the issue here...
/** Assign uploaded file to a variable */
$inputFileName = $_FILES['import']['tmp_name'];

/**  Identify the type of $inputFileName  **/
$inputFileType = \PhpOffice\PhpSpreadsheet\IOFactory::identify($inputFileName); /** throws error **/
/**  Create a new Reader of the type that has been identified  **/
$reader = \PhpOffice\PhpSpreadsheet\IOFactory::createReader($inputFileType); /** throws error **/



### Which versions of PhpSpreadsheet and PHP are affected?
^1.8 and 8.0

**Comment by oleibman** [comment]
This was fixed in October 2020 by PR #1625, which was included in release 1.15.0.

**Comment by TheRealMajaba** [comment]
The fix doesn't seem to be included in the 1.8.2 that you get with a normal composer update

**Comment by oleibman** [comment]
1.8.2 is a very old release - it dates back to July 2019. 1.15.0, which fixed the problem, was over a year later. Is there something that prevents you from moving to the current released version (1.17.1) or master?

**Comment by TheRealMajaba** [comment]
Must be some sort of problem with composer then, since composer update always updates me to 1.8.2. Thank you for the help
